homeassistant/components/overkiz/binary_sensor.py

```diff
@@ -105,6 +105,22 @@ class OverkizBinarySensorDescription(BinarySensorEntityDescription):
         )
         == 1,
     ),
+    OverkizBinarySensorDescription(
+        key=OverkizState.CORE_HEATING_STATUS,
+        name="Heating Status",
```

Comment:
Same as other PR:
```suggestion
        name="Heating status",
```

([It's specified](https://developers.home-assistant.io/blog/2022/07/10/entity_naming/) that "Device, Area, and Entity names all start with a capital letter, the rest of the words are lower case")